test/fixtures/BootstrapCommand/version-mismatch/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "independent"
```

Comment:
I know this is copypasta, but let's at least customize the name to match the directory.